File: Zomato.DomainModel/Models/OrderDetail.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Zomato.DomainModel.Models
{
    public class OrderDetail
    {
        public int OrderId { get; set; }
        public string Date { get; set; }
        public string RestaurantName { get; set; }
        public string UserName { get; set; }
        public string UserNumber { get; set; }
        public string UserEmail { get; set; }
        public string DeliveryLocation { get; set; }
        public int TotalAmount { get; set; }
        public List<ItemDetail> ItemDetail { get; set; }

        public OrderDetail()
        {
            ItemDetail = new List<ItemDetail>();
        }
    }
}

File: Zomato.Web/app/src/app/model/order-detail.ts
import { OrderItemDetail } from './order-item-detail';

export class OrderDetail {
  OrderId: number = 0;
  Date: string = null;
  RestaurantName: string = null;
  UserName: string = null;
  UserNumber: string = null;
  UserEmail: string = null;
  DeliveryLocation: string = null;
  TotalAmount: number = 0;
  itemDetail: OrderItemDetail[] = [];
}
